javascript/main.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+/* Book Class */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+/* Class UI : Handles UI Tasks */
+
+class UItask {
+  /* Show Books */
+  static displayBooks() {
+    const books = Storage.getBooks();
+    books.forEach((book) => {
+      UItask.addBook(book);
+    });
+  }
+
+  /* Add Book to list */
+  static addBook(book) {
+    const bookList = document.getElementById('book-list');
+    const bookContent = document.createElement('div');
+    bookContent.className = 'book';
+    bookContent.innerHTML = `<p class="book-title">"${book.title}"</p>
+                      <p class="book-author">by ${book.author}</p>
+                      <button class="delete">Remove</button>`;
+    bookList.appendChild(bookContent);
+  }
+
+  /* Delete Book From List */
+  static deleteBook(book) {
+    if (book.classList.contains('delete')) {
+      book.parentElement.remove();
+    }
+  }
+
+  /* Clear Inputs */
+  static clearInputs() {
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+  }
+}
+
+/* Class Store: Used to work with Local Storage */
+class Storage {
+  /* Loading books from local storage */
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  /* Add book to local Storage */
+  static addBookLocalStorage(book) {
+    const books = Storage.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  /* Remove book from local storage */
+  static removeBook(title) {
+    const books = Storage.getBooks();
+
+    books.forEach((book, i) => {
+      if (book.title === title) {
+        books.splice(i, 1);
```

Comment:
- [ ] [OPTIONAl] It is preferrable to use the array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.